content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,484 @@
+~~~~~~~~~~~~~~~~
+
+If managers want to leave notes that are only visible to other managers, enter them in the
+:guilabel:`Private Note` tab. The employee that is being evaluated does not have access to this tab;
+the tab does not appear on their appraisal.
+
+Schedule a meeting
+------------------
+
+Once both portions of an appraisal are completed (the :ref:`employee
+<appraisals/employee-feedback>` and :ref:`manager <appraisals/manager-feedback>` feedback sections)
+it is time for the employee and manager to meet, to discuss the appraisal. A meeting can be
+scheduled in one of two ways, either from the *Appraisals* application dashboard, or from an
+individual appraisal card.
+
+First, navigate to the main *Appraisals* application dashboard by going to
+:menuselection:`Appraisals application --> Appraisals`. Click on the activity icon beneath the
+appraisal date on the desired appraisal card and an activity summary pop-up appears. Click
+:guilabel:`+ Schedule an activity` and a blank activity form appears. Select :guilabel:`Meeting` for
+the :guilabel:`Activity Type` from the drop-down menu. The form changes so only the
+:guilabel:`Activity Type` and :guilabel:`Summary` fields appear. If scheduling an activity other
+than a meeting (such as a phone call or a to-do), the fields that appear on the schedule activity
+form changes.
+
+.. note::
+   The activity icon may appear as a number of different icons, depending on what (if any) scheduled
+   activities are in place, and their due dates. If no activities are scheduled, the icon is a gray
+   clock. For example, the icon may appear as a phone (for a phone call), a cluster of people (for a
+   meeting), a Gantt chart (for an appraisal form to be filled out), among others. The color of the
+   icon indicates the status; a green icon means an activity is scheduled in the future, a red icon
+   means the activity is past due. For more details on activities, refer to the
+   :doc:`../productivity/discuss/overview/plan_activities` document
+
+   .. image:: appraisals/activity-icons.png
+      :align: center
+      :alt: The appraisal cards with the various activity icons highlighted.
+
+Enter a brief summary in the field, such as `Annual Appraisal for (Employee)`. Next, click the
+:guilabel:`Open Calendar` button and the calendar loads. Navigate to and double click on the date
+and time for the meeting, and a :guilabel:`New Event` form pops up. Make any modifications desired
+to the form, such as the start and end times, or the name of the meeting. Add the person being
+appraised in the :guilabel:`Attendees` section, and include anyone ese who should be in the meeting,
+if necessary. To make the meeting a video call instead of an in-person meeting, click :guilabel:`+
+Odoo meeting` and a :guilabel:`Videocall URL` link appears in the field. When done making changes to
+the form, click :guilabel:`Save & Close`. The meeting now appears on the calendar, and the invited
+parties are informed via email.
```

Comment:
```suggestion
parties are informed via email.
```
Will they only be informed via email, or will they be informed based on their internal Odoo notification settings?

Reply:
I checked with a few runbots, and you only get a notification the day of the meeting, So when I scheduled one a few days out, nothing appears. When I changed the meeting to one hour from the current time, a notification on the internal Odoo alerts appears (the clock in the top right) and you see the meeting is coming up today. So I'll leave this the way it is, since they only get an email from what I can tell.